### ISSUE TITLE
How could I replace torch.nn.BatchNorm with SyncBatchNorm?

### ISSUE THREAD

**Issue by hubutui** [zhanghang1989/PyTorch-Encoding#171]
I wonder could I somehow iterate over the model and replace the batchnorm with SyncBatchNorm. Any suggestion is appreciated.

**Comment by zhanghang1989** [comment]
There is a similar function in NVIDIA distributed syncbn repo:
https://github.com/NVIDIA/apex/blob/b2f63c48408e1110bb1c7ad7bf6310141da30616/apex/parallel/__init__.py#L21

**Comment by hubutui** [comment]
Yeah, thank you. This function helps a lot. Though there's little issue, see https://github.com/NVIDIA/apex/issues/166.
